lib/forge_sdk/util.ex

```diff
@@ -145,36 +143,25 @@ defmodule ForgeSdk.Util do
 
   @spec to_tether_address(String.t()) :: String.t()
   def to_tether_address(hash) do
-    <<pk_hash::binary-size(40), _::binary-size(24)>> = hash
-    AbtDid.pkhash_to_did(:tether, pk_hash, form: :short)
+    AbtDid.hash_to_did(:tether, hash, form: :short)
```

Comment:
with this change, the original logic might be changed a little bit, pk_hash was cut from 40 bytes to 20 bytes. @dingpl716 please confirm this is ok.

Reply:
As discussed, it should be 20 bytes or 40 chars